mobile/src/main/java/ch/epfl/sdp/mobile/application/authentication/AuthenticatedUser.kt

```diff
@@ -99,7 +101,7 @@ class AuthenticatedUser(
    */
   val following: Flow<List<Profile>> =
       firestore
-          .collection("users")
+          .collection(ProfileDocument.Collection)
           .whereArrayContains("followers", user.uid)
           .orderBy("name")
```

Comment:
Missing fields.

Reply:
Added in 715a39794e4fd9228eaaf11ad0bf8a1682f5edfd